htdocs/admin/company.php

```diff
@@ -51,6 +51,16 @@
 
 $error = 0;
 
+// Logo small - Variable by default
```

Comment:
To get consistent values in all app for size of images, you can use 
getDefaultImageSizes()

Reply:
Thanks @eldy, done. I didn't know this function